cmd/stack_import.go

```diff
@@ -104,14 +111,32 @@ func newStackImportCmd() *cobra.Command {
 						result = multierror.Append(result, errors.New(msg))
 					}
 				}
+
+				if res.Status != "" {
+					msg := fmt.Sprintf("repairing URN %s in invalid '%s' state", res.URN, res.Status)
```

Comment:
I think we want to wordsmith this a bit: something like "resource %s was interrupted during a %s during a previous update".

We might not even want to display anything here, and instead relegate these warnings/errors to the "start a preview/update" paths.